build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
         classpath 'com.google.gms:google-services:2.1.0'
     }
 }
+project.ext.fabricKey = "";
```

Comment:
Can you explain what this line is for?  The `fabricKey` property is only pertinent to the `release` builds of the Zulip android app, not all the projects.